solidity/contracts/converter/ConverterBase.sol

```diff
@@ -231,30 +213,32 @@ abstract contract ConverterBase is ConverterVersion, IConverter, TokenHolder, Co
     }
 
     /**
-     * @dev withdraws tokens held by the converter and sends them to an account
-     * can only be called by the owner
-     * note that reserve tokens can only be withdrawn by the owner while the converter is inactive
-     * unless the owner is the converter upgrader contract
+     * @dev transfers reserves to a new converter during an upgrade
+     * can only be called by the converter upgraded which should be set at its owner
```

Comment:
'can be called only' IMO, but @yudilevi and me disagree on that

Reply:
Not important right now...